### PR TITLE
Chore/docs

### PR DESCRIPTION
**Pull request by delagroove**
## Issue
links for docs are outdated (in readme and github templates)

## Solution
remove 404s and pointed to current docs and discord as needed

## Breaking changes
none

## Testing
1.  open a pull request and review default links
2. check readme

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 At Reaction Commerce, we're dedicated to the open source community. In fact, we've designed our entire platform and business to grow from the passion and creativity that an open source community ignites. We've already attracted a small, dedicated team of open source contributors, and there's always room for more.
 
-If you'd like to join us, check out our detailed [Contributing Guide](https://docs.reactioncommerce.com/reaction-docs/master/contributing-to-reaction).
+If you'd like to join us, let's meet in Discord [Developer Chat](https://discord.gg/Bwm63tBcQY).
```
Comment (Akarshit) [inline CONTRIBUTING.md]: I think we shouldn't have the Gitter link here. It should be some kind of contributing guide.

Comment (delagroove) [reply CONTRIBUTING.md]: yeah, that page is pending (working on that tomorrow), should we wait for this merge until that page is ready?

Comment (Akarshit) [reply CONTRIBUTING.md]: Yeah let's wait for that.

File: .github/issue_template.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-Are you looking for help with getting started on Reaction? Please visit our [Reaction documentation](https://docs.reactioncommerce.com/reaction-docs/master/getting-started-developing-with-reaction).
+Are you looking for help with getting started on Reaction? Please visit our [Reaction documentation](https://mailchimp.com/developer/open-commerce/docs/fundamentals/).
 
 ## Prerequisites
-* [ ] Are you running the latest version?
+* [ ] what version are you running?
```
Comment (delagroove) [inline .github/issue_template.md]: this fixes #158

File: CONTRIBUTING.md (hunk 1 above)
Comment (delagroove) [reply CONTRIBUTING.md]: this is fixed

File: .github/issue_template.md (hunk 2 above)
Comment (Akarshit) [reply .github/issue_template.md]: `What version`

**Review by Akarshit — CHANGES_REQUESTED** [review]

**Review by Akarshit — APPROVED** [review]

**Pull request merged**